app/helpers/heatmap_helper.rb

```diff
@@ -159,6 +160,7 @@ def self.top_taxons_details(
     candidate_taxons.values.sort_by { |taxon| -1.0 * taxon["max_aggregate_score"].to_f }
   end
 
+  # rubocop:disable Metrics/ParameterLists
```

Comment:
By coincidence, we were talking about this, it has a way to ignore keyword parameters: `https://rubocop.readthedocs.io/en/latest/cops_metrics/#metricsparameterlists`.
Can you try to convert this to keyword parameters and set `CountKeywordArgs` to false on the rubocop configuration?

Reply:
Done